maud/tests/misc.rs

```diff
@@ -130,3 +130,10 @@ fn prefer_render_over_display() {
         "&lt;display&gt;"
     );
 }
+
+#[test]
+fn default() {
+    use maud::{Markup, PreEscaped};
+    assert_eq!(Markup::default().0, String::from(""));
```

Comment:
Nit: `String` can be compared with `&str`. So this works:

```suggestion
    assert_eq!(Markup::default().0, "");
```